packages/cli/README.md

```diff
@@ -0,0 +1,9 @@
+# @bull-monitor/cli
+
+Command line interface fpr adapter for [bull-monitor](https://github.com/s-r-x/bull-monitor)
+
+## Usage
+
+```sh
+npm i @bull-monitor/cli
```

Comment:
some usage example would be awesome, e.g. 
```sh
cmd-name -q my-queue
```
or just simply paste  commander --help output, but only with the command name

Reply:
Makes sense, updated it.